TalkAloud/TalkAloud/ViewControllers/AudioPlayerViewController.swift

```diff
@@ -78,7 +78,10 @@ class AudioPlayerViewController: UIViewController, AudioEngineStateChangeDelegat
     
     @IBAction func recordAudioButtonAction(_ sender: UIButton) {
         if AudioEngine.sharedInstance.audioState == .paused || AudioEngine.sharedInstance.audioState == .stopped {
-            AudioEngine.sharedInstance.setupRecorder(fileURL: AudioManager.sharedInstance.getNewRecordingURL())
+            // Use value forKey here to pass the url from the AudioRecording Object
```

Comment:
What do you mean with this comment?

Reply:
I forgot I made this but I was thinking at first that I was going to have to use the url property from CoreData. I thought I deleted this maybe this was from that commit I went back to.